fleet_vehicle_inspection/views/fleet_vehicle_inspection.xml

```diff
@@ -43,6 +43,7 @@
 					<group>
 						<group>
 							<field name="vehicle_id"/>
+                            <field name="driver_id"/>
```

Comment:
It seems this file has mixed tabs & spaces indent

Reply:
I think now it is right .. if it is the case I can pass the entire file in an xml formatter and already correct the entire file in this PR.

Reply:
Hello @marcelsavegnago This file still has tabs instead of spaces. Please fix this :)

Reply:
@mamcode check again please.